docs/x509.rst

```diff
@@ -964,6 +967,15 @@ Exceptions
 
         Returns the OID.
 
+.. class:: UnsupportedGeneralNameType
+
+    This is raised when a certificate contains an unsupported general name
+    type in an extension.
+
+    .. attribute:: type
+
+        :type: :term:`text` or int
```

Comment:
Also, just so I'm clear, the integer values are something out of the RFC, not OpenSSL's invention, right?